### DISCUSSION TITLE
How do I sort folders, subfolders and files like that in cherrytree

### DISCUSSION THREAD

**Discussion by raezyhsn** [SebastianMC/obsidian-custom-sort#76] [Q&A]
I'm new to Obsidian, so please excuse me if this question sounds stupid. 
I'm trying to organize the following layout

![b091f0aefe808b2f593ffe66a3e287d99bc5da4a](https://user-images.githubusercontent.com/115896549/234562430-1ade51bd-a4c7-41d8-96fb-c361a090e9c5.png)

Into the order in which they were created:

1. Open Redirect
- What is + examples

2. XSS
- What is
- Lab 1
- Lab 2
- Lab 3
- Lab 4

3. CSRF
- What is

4. IDOR
- What is
- Lab 1

5. LFD
- What is

I just want the folders, subfolders, notes and files to be sorted in the order in which they were created; Just like in cherrytree. I checked out Obsidian because: I love the UI and I find the Markdown editing much simpler and easier.

I have already tried the following sortspec configs:

```
---
sorting-spec: |
  order-desc: advanced created
---
```

Result:
![r](https://user-images.githubusercontent.com/115896549/234567575-37dda9eb-52c7-449a-b11b-6fbccd2d51af.png)

```
---
sorting-spec: |
  target-folder: /*
  order-desc: advanced created
---
```

Result:
![image](https://user-images.githubusercontent.com/115896549/234567789-08683521-5340-4ef5-b705-9eaba64200f0.png)

```
---
sorting-spec: |
 target-folder: //*
 /:files
  < created
 /folders
  < created
---
```

Result:
![image](https://user-images.githubusercontent.com/115896549/234567947-c0240702-d5f4-4c6b-a4f4-30728b4dfad5.png)

Somebody please help..

**Comment by raezyhsn** [comment]
I was able to achieve the result using the following:

```
---
sorting-spec: |
    target-folder: /*
    > advanced modified
---
```

**Reply by raezyhsn** [reply]
But if I add something to my notes and restart obsidian, the edited file is pushed to last in that folder.  I'm using linux btw. I just want the folders and notes to appear in the order in which they were created :/

**Comment by SebastianMC** [comment]
Hi @raezyhsn 

first and foremost, please make sure that you use the latest ([1.7.2](https://github.com/SebastianMC/obsidian-custom-sort/releases/tag/1.7.2)) release of the `Custom Sort` plugin. This release was published last month and it contains fixes for the exact sorting method which you need.

The sorting specification to use is close to what you supposed, namely:
```yaml
---
sorting-spec: |
    target-folder: /*
    < advanced created
---
```

Specifying date-based sorting can be confusing with regards to the order, ascending or descending. Dates are compared as numbers, a later timestamp is greater than an earlier one. The syntax resembles that: `<` means items go from the smaller to the larger - in human terms: from older to newer. This is exactly the ordering which you want, as I understand.

The `target-folder: /*` tells to apply the sorting specification to the root folder of the vault and all its subfolders.

In case the plugin update in connection with the above sorting spec doesn't work, please let me know
